classes/local/userstats_table.php

```diff
@@ -91,26 +91,28 @@ public function __construct(
         $tenantfield = get_config('local_ai_manager', 'tenantcolumn');
         if (!empty($purpose)) {
             $fields = 'u.id as id, lastname, firstname, locked, COUNT(value) AS requestcount, SUM(value) AS currentusage';
-            $from = '{local_ai_manager_request_log} rl LEFT JOIN {local_ai_manager_userinfo} ui ON rl.userid = ui.userid'
-                    . ' JOIN {user} u ON u.id = rl.userid';
-            $where = $tenantfield . ' = :tenant AND purpose = :purpose GROUP BY u.id';
+            $from = '{local_ai_manager_request_log} rl '
+                    . 'LEFT JOIN {local_ai_manager_userinfo} ui ON rl.userid = ui.userid '
+                    . 'JOIN {user} u ON u.id = rl.userid';
+            $where = $tenantfield . ' = :tenant AND purpose = :purpose GROUP BY u.id ORDER BY lastname';
             $params = [
                     'tenant' => $tenant->get_sql_identifier(),
                     'purpose' => $purpose,
             ];
             $this->set_count_sql(
-                    "SELECT COUNT(DISTINCT userid) FROM {local_ai_manager_request_log} rl JOIN {user} u ON rl.userid = u.id "
+                    "SELECT COUNT(DISTINCT rl.userid) FROM {local_ai_manager_request_log} rl "
```

Comment:
Maybe also use the variable $from here? Would make the code clearer.

Reply:
Yep.